src/components/RadioBorderedBox/index.tsx

```diff
@@ -37,102 +40,128 @@ const StyledBorderedBox = styled(BorderedBox)<BorderedBoxTypes>`
     if (error)
       return `
         border: 1px solid ${colors.danger.borderWeak} !important;
-        box-shadow: ${shadows.focusDanger};
       `
 
     if (checked)
       return `
         border: 1px solid ${colors.primary.borderWeak} !important;
-        box-shadow: ${shadows.focusPrimary};
       `
 
     return null
-  }}
+  }};
 `
 
 const StyledRadioContainer = styled.div`
   display: flex;
   align-items: center;
   gap: 0 ${({ theme }) => theme.space['1']};
-  margin-bottom: ${({ theme }) => theme.space['1']};
+`
+
+const StyledSpaceContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
 `
 
 const StyledError = styled.div`
   font-size: 12px;
   color: ${({ theme }) => theme.colors.danger.text};
-  padding: ${({ theme }) => `0 ${theme.space['0.5']}`};
 `
 
-const StyledExpandable = styled(Expandable)`
-  margin-top: ${({ theme }) => theme.space['1']};
-  margin-bottom: ${({ theme }) => theme.space['1']};
+const StyledMainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.space['1']};
 `
 
 type RadioBorderedBoxProps = RadioProps & {
   badgeSize?: ComponentProps<typeof Badge>['size']
   badgeText?: string
+  badgeProminence?: ComponentProps<typeof Badge>['prominence']
   badgeVariant?: ComponentProps<typeof Badge>['variant']
   children: ReactNode
   label: string
-  labelDescription?: ReactNode
+  labelDescription?: string
   name: string
-  size?: number
+  sideText?: string
   value: string | number
 }
 
 const RadioBorderedBox = ({
   label,
   labelDescription,
+  sideText,
   badgeText,
   badgeSize = 'small',
   badgeVariant = 'info',
+  badgeProminence,
   checked = false,
   onChange,
   onFocus,
   onBlur,
   disabled = false,
   name,
   value,
-  size = 24,
   children,
   error,
 }: RadioBorderedBoxProps) => (
-  <>
+  <StyledMainContainer>
     <StyledBorderedBox disabled={disabled} checked={checked} error={error}>
-      <StyledRadioContainer>
-        <Radio
-          name={name}
-          checked={checked}
-          onChange={onChange}
-          onFocus={onFocus}
-          onBlur={onBlur}
-          disabled={disabled}
-          value={value}
-          size={size}
-          error={error}
-        >
-          {label}
-        </Radio>
-        {labelDescription ? (
-          <Typography as="span">{labelDescription}</Typography>
+      <StyledSpaceContainer>
+        <StyledRadioContainer>
+          <Radio
+            name={name}
+            checked={checked}
+            onChange={onChange}
+            onFocus={onFocus}
+            onBlur={onBlur}
+            disabled={disabled}
+            value={value}
+            error={error}
+          >
+            {label}
+          </Radio>
+          {labelDescription ? (
+            <Text as="span" variant="body" disabled={disabled}>
+              {labelDescription}
+            </Text>
+          ) : null}
+          {badgeText && (
+            <>
+              &nbsp;
```

Comment:
do we still need that or can we remove fragment and &nbsp;

Checking design it seems that StyledRadioContainer should have a gap of 2 spaceUnit ? but it strange cause labelDescription may be a part or radio for DesignTeam ?